pkg/tsdb/cloudmonitoring/time_series_filter_test.go

```diff
@@ -386,6 +386,41 @@ func TestTimeSeriesFilter(t *testing.T) {
 		require.True(t, ok)
 		assert.Equal(t, "114250375703598695", labels["resource.label.instance_id"])
 	})
+
+	t.Run("Parse labels for distribution", func(t *testing.T) {
+		data, err := loadTestFile("./test-data/3-series-response-distribution-exponential.json")
+		require.NoError(t, err)
+		assert.Equal(t, 1, len(data.TimeSeries))
+		res := &backend.DataResponse{}
+		query := &cloudMonitoringTimeSeriesFilter{Params: url.Values{}}
+		err = query.parseResponse(res, data, "test_query")
+		require.NoError(t, err)
+		frames := res.Frames
+		assert.Equal(t, "test_query", frames[0].Meta.ExecutedQueryString)
+		custom, ok := frames[0].Meta.Custom.(map[string]interface{})
+		require.True(t, ok)
+		labels, ok := custom["labels"].(map[string]string)
+		require.True(t, ok)
+		assert.Equal(t, "grafana-prod", labels["resource.label.project_id"])
+	})
+
+	t.Run("Parse labels for distribution without points", func(t *testing.T) {
```

Comment:
Again, if I understood correctly and the meta data is being added, would you modify this test so it asserts the entire Meta object?